### PR TITLE
add rate limit information headers to api calls

### PR DESCRIPTION
**Pull request by bippum**

### CODE REVIEW

**Comment by howardchung** [comment]
@albertcui do you want to review this?

**Comment by howardchung** [comment]
I believe these failures are due to Travis CI not setting the test stripe credential.  @albertcui can confirm.  possibly we should just hardcode in the test stripe key if it's not sensitive.

File: svc/web.js [hunk 1]
```diff
@@ -139,6 +139,13 @@ app.use((req, res, cb) => {
...
 
   multi.exec((err, resp) => {
+    res.set({
+      'X-Rate-Limit': rateLimit,
+      'X-Rate-Limit-Remaining': rateLimit - resp[0],
+    });
+    if (!res.locals.isAPIRequest) {
+      res.set('X-Free-Requests-Remaining', config.API_FREE_LIMIT - Number(resp[2]));
+    }
     if (err) {
```
Comment (albertcui) [inline svc/web.js]: We should set the header after the err check. Otherwise, if there was an error, the resp information won't be accurate.

**Comment by albertcui** [comment]
Thanks! Merging.

**Pull request merged**